keyboards/lily58/keymaps/drasbeck/keymap.c

```diff
@@ -0,0 +1,188 @@
+/* Attribution-ShareAlike 4.0 International (CC BY-SA 4.0) 2020 Max Drasbeck 
+ *
+ * You are free to:
+ *
+ *  Share — copy and redistribute the material in any medium or format
+ *  Adapt — remix, transform, and build upon the material
+ *   for any purpose, even commercially.
+ *
+ * The licensor cannot revoke these freedoms as long as you follow the license terms.
+ */
+
+#include QMK_KEYBOARD_H
+
+#ifdef PROTOCOL_LUFA
+  #include "lufa.h"
+  #include "split_util.h"
+#endif
+#ifdef SSD1306OLED
+  #include "ssd1306.h"
+#endif
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
```

Comment:
None of these should be needed, as they should be included in the keyboard's header file. 

```suggestion
```